.env.template

```diff
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+
+
+TRELLO_API_KEY=PUT-TRELLO-API-KEY_HERE
+
+TRELLO_API_TOKEN=PUT-TRELLO-TOKEN_HERE
```

Comment:
Thanks for the clear instructions here!
One nice one to add might be your `TRELLO_BOARD_NAME` - you've hardcoded `'Welcome Board'` in `trello_items.py` which works fine but places some limits on adding new environments and might be a set-up subtlety that people would miss